### PR TITLE
added Minecraft Provider

### PR DESCRIPTION
**Pull request by Katzen48**
This is in response to #585 and based on the Microsoft-Live provider

### CODE REVIEW

**Comment by thinkverse** [comment]
This PR is missing the last step in adding a new provider.
> Add the provider to the `monorepo-builder.yml` subtree split config in the repository root.

**Comment by Katzen48** [comment]
@thinkverse done

**Comment by lucasmichot** [comment]
Hi @Katzen48 can you point me out to the official Oauth documentation ?

**Comment by Katzen48** [comment]
@lucasmichot there are two documentations, that summarized the flow:
- [https://mojang-api-docs.netlify.app/authentication/msa.html#oauth-2](https://mojang-api-docs.netlify.app/authentication/msa.html#oauth-2)
- [https://wiki.vg/Microsoft_Authentication_Scheme](https://wiki.vg/Microsoft_Authentication_Scheme)

**Comment by Katzen48** [comment]
@lucasmichot did you have time to verify this?
The pull request has been opened, more than 2 weeks ago.

**Comment by lucasmichot** [comment]
> @lucasmichot did you have time to verify this? The pull request has been opened, more than 2 weeks ago.

I guess we are all also busy outside open-source contribution @Katzen48 

Could you review https://github.com/Katzen48/Providers/pull/1 ?
This adjustment PR has been done 17 days ago

**Comment by Katzen48** [comment]
@lucasmichot sry, I got no notification, that you created a pr.
I accepted it and applied the style ci patch.

**Comment by Katzen48** [comment]
Done

**Review by lucasmichot — APPROVED** [review]

**Comment by Katzen48** [comment]
@lucasmichot any further changes, that need to be made?

File: src/Minecraft/composer.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Katzen48 | Tobi",
+            "email": "admin@katzen48.de"
+        }
+    ],
+    "require": {
+        "php": "^7.2 || ^8.0",
```
Comment (atymic) [inline src/Minecraft/composer.json:21]: drop less than 7.4 please

**Review by atymic — APPROVED** [review]
LGTM.

File: src/Minecraft/composer.json (hunk 1 above)
Comment (Katzen48) [reply src/Minecraft/composer.json]: Done.

**Comment by Katzen48** [comment]
I did the requested change, to drop the support for PHP less than 7.4

**Review by atymic — APPROVED** [review]

**Comment by Katzen48** [comment]
Thanks

**Pull request merged**